src/main/java/com/sfuentes/Stationery.java

```diff
@@ -0,0 +1,50 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Stationery {
+
+  Map<String, Integer> requested;
+
+  public Stationery() {
+    this.requested = new HashMap<>();
+  }
+
+  public void addRequest(String typeOfPrinting, int numOfCopies) {
+    if (requested.containsKey(typeOfPrinting))
+      requested.put(typeOfPrinting, requested.get(typeOfPrinting) + numOfCopies);
+    else
+      requested.put(typeOfPrinting, numOfCopies);
```

Comment:
This is just a comment, don't need to change.

this is another way to do it
requested.merge(typeOfPrinting,numOfCopies,Integer::sum);

you can read more about how it works:
https://stackoverflow.com/a/42648785/13158689

Reply:
fixed